README.md

```diff
@@ -93,7 +93,8 @@ This repository has the following structure:
   ├── pages                       # Website's pages source. It includes raw markdown files and React page templates.
   │    ├── about                  # Raw blog for /about page
   │    ├── blog                   # Blog posts
-  │    └── docs                   # Blog for /docs/* pages
+  │    ├── docs                   # Blog for /docs/* pages
+  |    └── tools                  # Blog for /tools/* pages
```

Comment:
```suggestion
  |    └── tools                  # Blog for /tools/* pages
```

@derberg I will like to have your concern here, as you know about the other tools page too. I personally don't think that the pages inside /tools/* are like blogs. They are more like providing necessary information and brief information about them in a light fashion, but not in too much descriptive way. Also, we have Tools Dashboard as part of it so I think we can describe it as "Various pages to describe tools".